src/npdmtool.c

```diff
@@ -400,20 +410,88 @@ int CreateNpdm(const char *json, void **dst, u32 *dst_size) {
         status = 0;
         goto NPDM_BUILD_END;
     }
+
+    cJSON_GetU64(fsaccess, "content_owner_id_min", &fac->CoiMin);
+    cJSON_GetU64(fsaccess, "content_owner_id_max", &fac->CoiMax);
+
+    cois = cJSON_GetObjectItemCaseSensitive(fsaccess, "content_owner_ids");
+    if (cJSON_IsArray(cois)) {
+        int idx = 0;
+        u64 *content_owner_id = (u64 *)((u8 *)fac + sizeof(FilesystemAccessControl));
+        cJSON_ArrayForEach(coi, cois) {
+            if (!cJSON_GetU64FromObjectValue(coi, content_owner_id)) {
+                status = 0;
+                goto NPDM_BUILD_END;
+            }
+            ++content_owner_id;
+            ++idx;
+        }
+        fac->CoiCount = idx;
+    }
+
+    cJSON_GetU64(fsaccess, "save_data_owner_id_min", &fac->SdoiMin);
+    cJSON_GetU64(fsaccess, "save_data_owner_id_max", &fac->SdoiMax);
+
+    sdois = cJSON_GetObjectItemCaseSensitive(fsaccess, "save_data_owner_ids");
+    if (cJSON_IsArray(sdois)) {
+        int idx = 0;
+        u64 *save_data_owner_id = (u64 *)((u8 *)fac + sizeof(FilesystemAccessControl));
```

Comment:
This is identical to the above content line... Won't this break if both content-owner/save-owner are specified?